redis/test_test.go

```diff
@@ -197,15 +198,22 @@ func DefaultServerAddr() (string, error) {
 // DialDefaultServer starts the test server if not already started and dials a
 // connection to the server.
 func DialDefaultServer(options ...DialOption) (Conn, error) {
+	return DialDefaultServerContext(context.Background(), options...)
+}
+
+// DialDefaultServerContext starts the test server if not already started and
+// dials a connection to the server. The provided context is used for the dial
+// and for the initial FLUSHDB command.
```

Comment:
Lets avoid describing internal detail, so we don't have to update if this changes.
```suggestion
// DialDefaultServerContext starts the test server if not already started and
// dials a connection to the server.
```